public/editor-client/src/api/index.ts

```diff
@@ -1206,3 +1216,97 @@ export const updateGlobalBlocks = async (
 };
 
 //#endregion
+
+//#region Default Templates
+export const getDefaultLayouts = async (
+  url: string
+): Promise<{ templates: LayoutsAPI[]; categories: Record<string, string> }> => {
+  const res = await fetch(`${url}/get-layouts-chunk`).then((r) => r.json());
+
+  return { templates: res.collections, categories: res.categories };
+};
+
+export const getDefaultLayoutsPages = async (
+  url: string,
+  id: string
+): Promise<{
+  collections: LayoutsPageAPI[];
+  paginationInfo: {
+    itemsPerPage: number;
+    lastPage: number;
+    totalCount: number;
+  };
+  styles: Style;
+}> => {
+  const data = await fetch(
+    `${url}/get-layouts-pages?project_id=${id}&per_page=20`
+  ).then((r) => r.json());
+
+  return data;
+};
+
+export const getDefaultLayoutData = async (
+  url: string,
+  layoutId: Literal,
+  id: string
+): Promise<{
+  items: DefaultBlockWithID[];
+}> => {
+  const data = await fetch(
+    `${url}/get-layouts-page?project_id=${layoutId}&page_slug=${id}`
+  ).then((r) => r.json());
+
+  return Json.read(data[0].pageData) as {
+    items: DefaultBlockWithID[];
+  };
```

Comment:
remove casting, because of casting we don't handle some errors

as example here `Json.read` can return undefined 
![image](https://github.com/ThemeFuse/Brizy/assets/52622998/cc1f009d-f1bf-4cf9-8713-979f7af48943)

but you cast is as object with items inside